packages/devtools_app/lib/src/memory/memory_heap_tree_view.dart

```diff
@@ -367,18 +395,50 @@ class HeapTreeViewState extends State<HeapTree>
       padding: const EdgeInsets.only(top: denseRowSpacing),
       child: Column(
         children: [
+          const SizedBox(height: defaultSpacing),
           Row(
             children: [
-              _buildSnapshotControls(textTheme),
-              const Expanded(child: SizedBox(width: defaultSpacing)),
-              // TODO(peterdjlee): Implement filter and search functionality for treemap.
+              TabBar(
+                labelColor: themeData.textTheme.bodyText1.color,
+                isScrollable: true,
+                controller: tabController,
+                tabs: HeapTreeViewState.DartHeapTabs,
+              ),
+              const Expanded(child: SizedBox()),
```

Comment:
remove this and use MainAxisAlignment.spaceBetween on the row

Reply:
Done.